### ISSUE TITLE
Can we default a model/type to inlcude relations rather than splitting them?

### ISSUE THREAD

**Issue by gregg-cbs** [chrishoermann/zod-prisma-types#101]
Hi Chris,

Thanks for this package, it has saved us a lot of time!

I have noticed in the generated schema file you have chosen split the relation fields into another type, making the original type a lesser version of the prisma model which is confusing because one would expect the exported type to be the full model.

We have a product model in prisma and we expect to use `ProductSchema` but we have to use `productWithRelations`. Can you add an option to the generator config where we can choose to make `ProductSchema` the full model with relations?

**Comment by chrishoermann** [comment]
@gregg-cbs on the schemas with relations `z.ZodType<..>` is used which disallows `.pick`, `.omit` and other useful oject methods (as mentioned in #102). So I decided to implement the scalar and enum fields as a simple `z.object`, which allows the aformentioned methods, and later extend this schema with relation types, which disallow these methods. 
It is not ideal but it was the only solution that came to my mind that allows some flexibility, when I decided against the in #102 mentioned `satisfies` operator.
